pkg/tsdb/cloudwatch/get_metric_data_executor_test.go

```diff
@@ -12,9 +13,25 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGetMetricDataExecutorTest(t *testing.T) {
+func TestGetMetricDataExecutorTestRequest_should_round_up_endTime(t *testing.T) {
```

Comment:
total nit and not important but why camelcase and snake case together? I think if you want an "it should do whatever" type pattern that we have with the jest tests I think you can do something like:
```
func TestGetMetricDataExcecutor(t *testing.T) {
    t.Run("should do things", func (t *testing.T) {
        // whatever the first test is
    }
    t.Run("should do other things", func (t *testing.T) {
        // whatever the second test is
    }
}
```